transformers/hf_api.py

```diff
@@ -129,10 +130,13 @@ def presign_and_upload(self, token, filename, filepath):
         # Even though we presign with the correct content-type,
         # the client still has to specify it when uploading the file.
         with open(filepath, "rb") as f:
+            pf = ProgressFileReader(f)
```

Comment:
Maybe call it `TqdmProgressFileReader` if it's included in `file_utils`?

Reply:
renamed it, let's move it to file_utils if/when we need it somewhere else.